src/components/Icons/BookPen.svg

```diff
@@ -0,0 +1,14 @@
+
+<svg width="25px" height="20px" viewBox="0 0 25 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="Symbols" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Registrar-Icon">
+            <g>
+                <rect id="Rectangle-2" fill="#4B84FF" x="0" y="0" width="16" height="20" rx="3"></rect>
+                <path d="M20,15.8125 C20,15.928875 20.0525,16.04 20.1465,16.1218125 L22.1465,17.8718125 C22.244,17.957125 22.372,18 22.5,18 C22.628,18 22.756,17.957125 22.8535,17.8718125 L24.8535,16.1218125 C24.9475,16.04 25,15.928875 25,15.8125 L25,4 L20,4 L20,15.8125 Z" id="Fill-4" fill="#4B84FF"></path>
+                <path d="M22,0 L23,0 C24.1045695,-2.02906125e-16 25,0.8954305 25,2 L25,3 L20,3 L20,2 C20,0.8954305 20.8954305,2.02906125e-16 22,0 Z" id="Rectangle-6" fill="#4B84FF"></path>
+                <rect id="Rectangle-3" fill="#FFFFFF" transform="translate(6.000000, 5.000000) scale(1, -1) translate(-6.000000, -5.000000) " x="2" y="4" width="8" height="2" rx="1"></rect>
+                <rect id="Rectangle-3-Copy" fill="#FFFFFF" transform="translate(6.000000, 9.000000) scale(1, -1) translate(-6.000000, -9.000000) " x="2" y="8" width="8" height="2" rx="1"></rect>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
<img width="203" alt="screenshot 2019-02-14 at 05 35 43" src="https://user-images.githubusercontent.com/2630/52787246-7a42ec80-301a-11e9-92cb-03d72b04c585.png">

Can you get rid of the end of the line thing (I think you have to add new line below)